packages/ember/README.md

```diff
@@ -82,13 +83,53 @@ you can import `instrumentRoutePerformance` and wrap your route with it.
 import Route from '@ember/routing/route';
 import { instrumentRoutePerformance } from '@sentry/ember';
 
-export default instrumentRoutePerformance(
-  class MyRoute extends Route {
-    model() {
-      //...
-    }
+class MyRoute extends Route {
+  model() {
+    //...
   }
-);
+}
+
+export default instrumentRoutePerformance(SlowLoadingRoute);
```

Comment:
This `SlowLoadingRoute` looks mysterious here as the code sample could be better to make it more clear what that is.

Reply:
Not only is it mysterious, it's also wrong 😅, nice catch 👍